css-two-column-layout/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Two Column Layout</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="header">
+          <h1>SCALE, SPEED, AGILITY</h1>
+        </div>
+      </div>
+      <div class="row">
+        <div class="headp">
+          <p>
+            We've dedicated thousands of man hours to making healthcare
+            institutions in the Helium network run 10x more efficiently and
+            deliver an unparalleled quality of care.
+          </p>
+        </div>
+      </div>
+      <div class="row">
+        <div class="imgcolumn">
```

Comment:
so the only thing I would add here is a `column-half` wrapper which would be the parent of the `imgcolumn` and `column`  classes. as this is a `two column` design, it is best for the main columns to be one half of the total row, and this also encapsulates the content nicely for each collection of elements in a column.